_pages/opportunity-grants.md

```diff
@@ -1,86 +1,71 @@
 
-### When will I receive my opportunity grant?
 
-The opportunity grant is paid to the recipient at the conference or shortly after.
+*under-represented or marginalised groups  include, but are not limited to:
+- women and other gender minorities of all expressions and identities; e.g. trans, agender and non-binary people
+- people of colour
+- sexuality minorities, including asexual people
+- people with disabilities, whether visible and invisible
+- neurodivergent people
+- people with chronic illnesses or diseases
+- religious and ethnic minorities
+- age minorities (under ~21, over ~60)
+- people experiencing poverty
+- homeless and home/food-insecure people
+- caregivers of children or other dependents
+- people who have experienced trauma and its aftermath (PTSD, anxiety, etc)
+- people living with or recovering from substance abuse
+- people who have been displaced or affected by wars in Ukraine and elsewhere
 
-### How will I receive my opportunity grant?
 
-The opportunity grant is paid to the recipient in a check for US account holders.
-A bank transfer is used for non-US account holders.
-We typically use TransferWise for international payments but have used other services for currencies they do not support.
+## How to apply
 
-We require receipts for the items that the opportunity grant was spent.
+Grant applications will open soon.
 
-### Is there anything I should know if I am traveling into the US from another country?
+You can apply for 3 types of grant:
 
-If you are traveling into the United States, we strongly encourage you to wait to purchase airfare or book a hotel room until after your visa has been accepted.
+- Conference ticket: a free standard ticket to the full conference talks and sprints (note that if you are a speaker, you will receive a free ticket)
+- Travel and accommodation: we will consider applications for contributions towards travel and accommodation costs up to a maximum £300
+- Visa application fee: reimbursement for the cost of a UK Standard Visitor visa (up to £100).
 
-You can request a visa letter by <a href="mailto:{{site.visa_email}}">emailing us</a>.
-
-If you are denied entry to the US with a valid visa, we will pay your non-refundable costs (up to the amount of your opportunity grant).
-
-### What if there is an emergency and I cannot attend?
+{% comment %}
+To apply, please fill out [this form](https://2023.djangocon.eu/opportunity-grants/).
 
-If you are denied entry to the US with a valid visa, we will pay your non-refundable costs (up to the amount of your opportunity grant).
-For other emergencies or additional costs, we will review the recipient's needs on a case-by-case basis.
+The closing date for grant applications is 17:00 (Europe/London) on 31st March 2023. We aim to inform you of the results within a few days after closing the application window. Note that if you need a grant, you must apply before the deadline.
```

Comment:
Would be nice to use a timezone conversion service for things like this – I took the liberty to add a link, doesn’t have to be this specific service if we’d prefer another one.

```suggestion
The closing date for grant applications is [17:00 (Europe/London) on 31st March 2023](https://www.timeanddate.com/worldclock/fixedtime.html?p1=304&iso=20230331T170000&msg=Closing%20of%20grant%20applications%20for%20DjangoCon%20Europe%202023). We aim to inform you of the results within a few days after closing the application window. Note that if you need a grant, you must apply before the deadline.
```